config/locales/views/en.yml

```
@@ -325,46 +325,59 @@ en:
   abuse_reports:
     new:
       heading:
-        page_title: Report Abuse
+        page_title: 'Policy Questions & Abuse Reports'
       purview:
-        about: The Abuse committee looks after all of your concerns regarding content and user behavior which breach the Archive %{tos_link}. For technical help, please %{support_link}.
-        tos: Terms of Service
-        support: contact Support
-      do_not_spam_html: <strong>We investigate every report we receive.</strong> For this reason, we ask you to not submit several reports regarding any one issue unless you have additional information to offer. We also ask that you do not encourage other users to submit a report regarding content you have already reported. Doing so slows down our response and reaction time considerably, as we are a small volunteer-based committee.
+        about: 'Please use this form for questions or reports about violations of the Archive’s %{tos_link}.'
+        technical_help: 'For technical help, or if you would like to manage your Fannish Next-of-Kin, please %{support_link}.'
+        dmca_takedown: 'If you would like to initiate a formal DMCA takedown request, please %{dmca_link}.'
+        tos: 'Terms of Service'
+        support: 'contact our Support team'
+        dmca: 'review our DMCA Policy and contact our Legal team'
       reportable:
-        intro: 'Please submit a report via this form if you have not submitted this report within the past sixty (60) days, and you:'
-        hack: believe that your account has been hacked, or that someone is trying to hack your account, or
-        tos: believe you have found content uploaded to the Archive which breaches our Terms of Service, or
-        dmca:
-          claim: would like to lodge a DMCA claim - please review our %{dmca_link}
-          policy: DMCA Policy
-      languages_html: <strong>We can answer Abuse reports in %{list}.</strong> Please allow for an additional delay for responses in any language other than English.
+        intro: 'Potential reasons to submit a report to the Policy & Abuse team:'
+        violation: 'You have encountered content on the Archive that violates the %{tos_link}.'
+        content: 'You would like to know whether specific content is allowed on the Archive.'
+        harassment: 'You have experienced or witnessed harassment on the Archive.'
+        hack: 'You believe that your account was hacked.'
+        suspended: Y'ou have been suspended or your work has been removed, and you have not received an email explaining why.'
```

A sneaky quote crept in one character too far right:
```suggestion
        suspended: 'You have been suspended or your work has been removed, and you have not received an email explaining why.'
```